lib/isochrone-tile.js

```diff
@@ -42,11 +42,11 @@ export default function isochroneTile (imageData, {height, scaleFactor, surface,
         val = surface[ypsurf * width + xpsurf]
       }
 
-      let colorScheme = colorScheme || function (val) {
+      const _colorScheme = colorScheme || function (val) {
```

Comment:
Agree with that. My main issue was that it appeared to be reassigning the parameter but it was just reusing parameter names. I'll change it to use an ES6 default.